annotator_store/models.py

```diff
@@ -10,25 +10,29 @@
 from django.contrib.auth.models import Group, User
 from django.utils.html import format_html
 from jsonfield import JSONField
-try:
-    import guardian
-    from guardian.shortcuts import assign_perm, get_objects_for_user, \
-        get_objects_for_group, get_perms_for_model, get_perms
-    from guardian.models import UserObjectPermission, GroupObjectPermission
-except ImportError:
-    guardian = None
-import six
-
 
+# get system logger for notifying test errors/warnings
 logger = logging.getLogger(__name__)
-
+#import settings to read ANNOTATION_OBJECT_PERMISSIONS flag
```

Comment:
These changes look good to me, but please clean up the whitespace (pep8: space after `#`, space after comma in getattr) and re-order so the comments are with and other imports aren't interspersed or connected with this.  Since this is now fully a conditional import, I think just but it in a separate block after all the other imports rather than mixing them.